cmd/ego/main.go

```diff
@@ -45,15 +50,37 @@ func main() {
 		fmt.Println()
 	case "run":
 		if len(args) > 0 {
-			os.Exit(cli.Run(args[0], args[1:]))
+			x, err := c.Run(args[0], args[1:])
+			if err != nil {
+				switch err {
+				case cli.ErrElfNoPie:
+					fmt.Println("ERROR: Binary could not be run by Open Enclave.")
+					fmt.Println("Possibly the binary was not build with 'ego-go build'?")
+				case cli.ErrValidAttr0:
+					fmt.Println("ERROR: Binary could not be run by Open Enclave.")
```

Comment:
```suggestion
					fmt.Println("ERROR: Binary could not be loaded.")
					fmt.Println("Possibly the binary was not build with 'ego-go build'?")
				case cli.ErrValidAttr0:
					fmt.Println("ERROR: Binary could not be loaded.")
```
I think we shouldn't mention OE here. It's actually ERT, but I think these internals are rather confusing than helpful.